NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -897,14 +897,16 @@ def get_event(self, run_nr, event_id):
     def end(self):
         if self.__counter:
             self.logger.info(
-                f"\n\tRead {self.__counter} events (skipped {self.__skipped} events, {self.__invalid} invalid events)"
+                f"\n\tRead {self.__counter} events ({self.__skipped} events are skipped (filtered), {self.__invalid} invalid events)"
                 f"\n\tTime to initialize data sets  : {self._time_begin:.2f}s"
                 f"\n\tTime to read all events       : {self._time_run:.2f}s"
                 f"\n\tTime to per event             : {self._time_run / self.__counter:.2f}s"
                 f"\n\tRead {self.__n_runs} runs, skipped {self.__skipped_runs} runs.")
         else:
-            self.logger.info(
-                f"\n\tRead {self.__counter} events   (skipped {self.__skipped} events, {self.__invalid} invalid events)")
+            self.logger.warning(
+                f"\n\tRead {self.__counter} events   (skipped {self.__skipped} events, {self.__invalid} invalid events)"
+                f"\n\tTime to initialize data sets  : {self._time_begin:.2f}s"
+                f"\n\tTime to read all events       : {self._time_run:.2f}s")
```

Comment:
This would probably be a good fit for the new 'status' logging level implemented in #602. But seeing as that's not merged yet, and this output only occurs on calling `end`, I think this is fine.